src/repl/scala/tools/nsc/interpreter/MemberHandlers.scala

```diff
@@ -136,20 +136,28 @@ trait MemberHandlers {
           if (mods.isLazy) codegenln(false, "<lazy>")
           else any2stringOf(path, maxStringElements)
 
-        val nameString = string2code(prettyName) + (if (showObjIds) s"""" + f"@$${System.identityHashCode($path)}%8x" + """" else "")
+        val varOrValOrLzy =
+          if (mods.isMutable) "var"
+          else if (mods.isLazy) "lazy val"
+          else "val"
```

Comment:
How does this compare to `symbol.keyString`?

Reply:
this would still require extra logic for lazy

Reply:
and also a `var` symbol outputs as `def`